builtin_regexp.go

```diff
@@ -235,12 +241,24 @@ func compileRegexp(patternStr, flags string) (p *regexpPattern, err error) {
 		patternStr = convertRegexpToUtf16(patternStr)
 	}
 
-	re2Str, err1 := parser.TransformRegExp(patternStr)
+	var (
+		re2Str string
+		err1   error
+	)
+	if !dotAll {
+		re2Str, err1 = parser.TransformRegExp(patternStr)
```

Comment:
You don't need to fallback to regexp2 to support dotAll, it should be possible to pass additional flag to `TransformRegExp` to make it not replace dots with Re2Dot.

Reply:
Ya, I have considered doing this, I can submit a version for you to take a look at.